appveyor.yml

```diff
@@ -26,7 +28,12 @@ install:
 - ps: C:\projects\spectre\scripts\InstallNode.ps1
 
 build_script:
-- cmd: ng build -env=%CONFIGURATION% --aot=false >nul 2>&1
+- cmd:
+    if(configuration == prod) {
+      ng build -env=%CONFIGURATION% prod_flag >nul 2>&1
```

Comment:
You've probably missed `%`. Since you are `if`ing this, you can just put `--target=production` here.